flow/table.go

```diff
@@ -153,6 +154,7 @@ func NewTable(updateHandler *Handler, expireHandler *Handler, nodeTID string, op
 	}
 
 	t.updateVersion = 0
+	t.removedFlows = 0
```

Comment:
this line is useless, in fact the upper one too ;)

Reply:
done for both